Tensile/Source/client/source/Reference.cpp

```diff
@@ -164,10 +163,10 @@ namespace Tensile
                 if(op.type == TensorOp::Type::ComplexConjugate)
                     bConjugate = true;
 
-            std::vector<size_t> freeASize(problem.freeIndicesA().size());
```

Comment:
cppcheck:
- Variable 'freeIndicesA' is assigned a value that is never used.
- Variable 'freeIndicesB' is assigned a value that is never used.
- Variable 'batchIndices' is assigned a value that is never used.